src/test/java/pageObjects/listingDetails_page.java

```diff
@@ -0,0 +1,74 @@
+package pageObjects;
+
+import static org.testng.Assert.assertTrue;
+
+import org.openqa.selenium.By;
+
+import org.openqa.selenium.WebDriver;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import stepDefinitions.hooks;
+
+
+public class listingDetails_page {
+	
+	private WebDriver driver;
+	private static WebElement element = null;
+	
+	
+	public static void load_listings_details_page(WebDriver driver) {
```

Comment:
Can this method take another parameter that is the ID of the listing? Then you could use this single method in many places, where your Feature is written as something like `Given I am on the Details page for Listing "8481"`